package.json

```diff
@@ -113,35 +113,35 @@
     "karma-super-dots-reporter": "0.1.0",
     "lazypipe": "1.0.1",
     "lolex": "2.3.1",
-    "markdown-link-check": "3.0.3",
-    "minimatch": "3.0.0",
+    "markdown-link-check": "3.1.5",
+    "minimatch": "3.0.4",
     "minimist": "1.2.0",
     "mocha": "4.0.1",
-    "moment": "2.18.1",
-    "morgan": "1.7.0",
-    "nodemon": "1.11.0",
+    "moment": "2.20.1",
+    "morgan": "1.9.0",
+    "nodemon": "1.14.0",
     "path": "0.12.7",
-    "phantomjs-prebuilt": "2.1.14",
+    "phantomjs-prebuilt": "2.1.16",
     "postcss": "6.0.14",
     "postcss-import": "11.0.0",
-    "pretty-bytes": "2.0.1",
+    "pretty-bytes": "4.0.2",
     "prop-types": "15.6.0",
     "react": "16.2.0",
     "react-addons-shallow-compare": "15.6.2",
-    "react-dates": "13.0.6",
+    "react-dates": "16.0.1",
```

Comment:
@cvializ should this be in dependencies and not devDependencies?

Reply:
Just merged this, but I think it should be moved to `dependencies`. I'll move it up in a follow up PR right now and revert it to its old 13.0.6 version, and update the documentation.

Reply:
It turns out that @cvializ added several packages to `package.json` in his date picker PR. I've started a new discussion / PR at #12571 to see if this is indeed a concern. It's possible that the only package that needs to move to `dependencies` is `react-dates`, since the only import from `node_modules` in `amp-date-picker` is here: https://github.com/ampproject/amphtml/blob/master/extensions/amp-date-picker/0.1/amp-date-picker.css

Let's continue this discussion in #12571.